config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -112,10 +128,10 @@ presubmits:
           resources:
             requests:
               cpu: "4"
-              memory: "4Gi"
+              memory: "8Gi"
             limits:
               cpu: "4"
-              memory: "4Gi"
+              memory: "8Gi"
```

Comment:
These can probably stay 4Gi. Since `k8s-infra-prow-build` already required resource quotas, we can assume 4Gi was sufficient.